recirq/qml_lfe/learn_states_q_test.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2021 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import os
+import numpy as np
+from . import learn_states_q
+
+
+def _predict_exp(data, pauli_list, two_copy):
```

Comment:
This function is also hard to understand.  Maybe a docstring here to explain it might help?

Reply:
Added a short docstring.